Base/Python/slicer/util.py

```diff
@@ -641,6 +641,43 @@ def array(pattern = "", index = 0):
 
 
 #
+# Qt
+#
+
+class QtWidgetMixin(object):
+  def __init__(self):
+    super(QtWidgetMixin, self).__init__()
+    self.Widget = None
+
+  def loadUI(self, path, fail_silently=False):
+    # Attempts to load and return a ui file that the given path points to.
+    # This method will raise an AssertionError if the ui file cannot be
+    # loaded and fail_silently is False (default).
+    import qt
+    qfile = qt.QFile(path)
+    qfile.open(qt.QFile.ReadOnly)
+    loader = qt.QUiLoader()
+    widget = loader.load(qfile)
+    if not fail_silently and not widget:
+      errorMessage = "Could not load UI file: " + str(path) + "\n\n"
+      raise AssertionError(errorMessage)
```

Comment:
The idea is that the user can decide what is in ```self.Widget``` since this is what is access in the ```get()``` method.
Most likely, all the users would do something like this:
```python
def MyClass(QtWidgetMixin):
  def __init__(self):
    super(MyClass, self).__init__()
    self.Wigdet = self.loadUI('/my_path/myWigdet.ui')

    myWidget = self.get('WidgetName')
```

But I could also see something like this:
```python
def MyClass(QtWidgetMixin):
  def __init__(self):
    super(MyClass, self).__init__()
    firstPart = self.loadUI('/my_path/firstPart.ui')
    secondPart = self.loadUI('/my_path/firstPart.ui')

    self.Wigdet = qt.QWidget()
    layout = qt.QVBoxLayout()
    layout.addWidget(firstPartWidget)
    layout.addWidget(secondPartWidget)
    self.Widget.setLayout(layout)

    w1 = self.get('WidgetName_FromFirstPart')
    w2 = self.get('WidgetName_FromSecondPart')
```